pkg/live/rgpath_test.go

```diff
@@ -127,6 +153,23 @@ func TestPathManifestReader_Read(t *testing.T) {
 			namespace:      "test-namespace",
 			expectedErrMsg: "unknown resource types: Custom.custom.io",
 		},
+		"local-config is filtered out": {
```

Comment:
orthogonal to this: I'm pretty much convince that declaring packages in Go like this is an anti-pattern. At some point, need to move away from this.